app/views/items/_new.html.erb

```diff
@@ -1,9 +1,12 @@
-<%= bootstrap_form_for @item, layout: :horizontal, label_col: "col-sm-4", url: new_ticket_path do |f| %>
+<%= bootstrap_form_for @item, layout: :horizontal, label_col: "col-sm-4", class: "item-form", url: new_ticket_path do |f| %>
   <%= f.collection_select :event_id, Event.active, :id, :title, prompt: "Select the correct event", control_col: "col-sm-8" %>
   <%= f.text_field :section, control_col: "col-sm-3" %>
   <%= f.text_field :row, control_col: "col-sm-3" %>
   <%= f.text_field :seat, control_col: "col-sm-3" %>
   <%= f.number_field :unit_price, control_col: "col-sm-4" %>
   <%= f.select :delivery_method, [["Electronic", "electronic"], ["Physical", "physical"]], prompt: "Select the delivery method", control_col: "col-sm-6" %>
+  <div class="item-form-ticket">
+    <%= f.file_field :ticket, control_col: "col-sm-8" %>
+  </div>
```

Comment:
There are probably a bunch of other places in our program we can do the same thing were we give something a class and then use Jquery to determine if it should be hidden or not. 

I like this! :sparkles: